src/components/SaveExport/ExportSelection.tsx

```diff
@@ -110,7 +113,7 @@ const ExportSelection = ({
         if (cursorBegin != null) {
             updateRadioSelected(2);
         } else {
-            updateRadioSelected(0);
+            updateRadioSelected(1);
```

Comment:
Referencing the options by numbers 0, 1, and 2 makes this a bit harder to read for me. So, in the future it might be nice to refactor this, e.g. to reference them by a name (not only here).